package.json

```diff
@@ -1,129 +1,139 @@
 {
-  "name": "hocs-management-ui",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "start": "node index.js",
-    "build-dev": "webpack --mode=development --env.NODE_ENV=development",
-    "build-prod": "webpack --mode=production --env.NODE_ENV=production",
-    "test": "jest --coverage",
-    "jest": "node_modules/jest/bin/jest.js  .",
-    "lint:js": "eslint ./ --ignore-path .gitignore",
-    "lint:jsx": "eslint ./ --ignore-path .gitignore",
-    "lint:ts": "tslint -c tslint.json 'src/**/*.ts'",
-    "lint:tsx": "tslint -c tslint.json 'src/**/*.tsx'",
-    "lint-staged": "node_modules/lint-staged/index.js",
-    "lint": "npm run lint:js & npm run lint:jsx & npm run lint:ts & npm run lint:tsx"
-  },
-  "repository": {
```

Comment:
```js
You might want to replace this with a more accurate link instead of removing it entirely.
{
  "repository": "github:UKHomeOffice/hocs-management-ui"
}
```

Reply:
agreed - doesn't hurt to have this in